src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -96,41 +138,97 @@ def export_config(cmd,
                   dest_connection_string=None,
                   dest_label=None,
                   # to-app-service parameters
-                  appservice_account=None):
-    # fetch key values from user's configstore
-    src_kvs = __read_kv_from_config_store(
-        cmd, name=name, connection_string=connection_string, key=key, label=label, prefix_to_remove=prefix)
-
-    # if customer needs preview & confirmation
-    if not yes:
-        # fetch key values from destination
-        if destination == 'file':
-            dest_kvs = []  # treat as empty file and overwrite exported key values
-        elif destination == 'appconfig':
+                  appservice_account=None,
+                  skip_features=False):
+    src_features = []
+    dest_features = []
+    dest_kvs = []
+    destination = destination.lower()
+
+    if skip_features and not yes:
```

Comment:
>if skip_features and not yes: [](start = 4, length = 29)

the logic here can be optimized to avoid double negative check making codes hard to read. 
it can be something like below

```
# first get everything we need from source App Configuration.

__read_kv_from_config_store   3# fetch all key values 
__discard_features_from_retrieved_kv

if not skip_features:
    # fetch all features

# second check if we need preview
if not yes:
    # fetch all key values excluding features from destinations, "AppConfig" or "AppService"
    if not skip_feature:
         # fetch all features from destination, "AppConfig"
    #print preview and ask for confirmation 
# third export to destination

```